src/server/qtquick/private/wquickswipehandler_p.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2023 tjd <tanjingdong@uniontech.com>.
+// SPDX-License-Identifier: Apache-2.0 OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+
+#pragma once
+
+#include <wglobal.h>
+#include <private/qquickmultipointhandler_p.h>
+
+
+WAYLIB_SERVER_BEGIN_NAMESPACE
+
+
+class WAYLIB_SERVER_EXPORT WQuickSwipeHandler : public QQuickMultiPointHandler
```

Comment:
No, This class shouldn't be placed in the waylib project, this functionality isn't what waylib should provide. If the compositor needs it, you can add it in your compositor project.